infra/api.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/python3
 
 import bottle
+from bottle import response
```

Comment:
why not `bottle.response` below? is this import even used?

Reply:
Fixed it; the language server snuck this import in when I was doing local testing. I've removed it.